README.md

```diff
@@ -6,8 +6,9 @@ This project is for creating automated tests for verifying the functionality of
 | IDE           | Version |
 | ------------- | ------------- |
 | Redhat Code Ready Studio  | 12.18.0.GA  |
-| Eclipse IDE   | 2020-12  |
+| Eclipse IDE for J2EE developers  | 2020-12  |
```

Comment:
We are testing 2020-09 only for Eclipse

Reply:
I have tested with this version (2020-12), still need to update?

Reply:
Docs mention 2020-09 , if we upgrade here we need to upgrade everywhere
.so let's keep it the same for now